zebrad/src/components/mempool/pending_outputs.rs

```diff
@@ -0,0 +1,63 @@
+//! Pending [`transparent::Output`] tracker for [`AwaitOutput` requests](zebra_node_services::mempool::Request::AwaitOutput).
+
+use std::{collections::HashMap, future::Future};
+
+use tokio::sync::broadcast;
+
+use tower::BoxError;
+use zebra_chain::transparent;
+
+use zebra_node_services::mempool::Response;
+
+#[derive(Debug, Default)]
+pub struct PendingOutputs(HashMap<transparent::OutPoint, broadcast::Sender<transparent::Output>>);
```

Comment:
Missing docs

Reply:
Added in https://github.com/ZcashFoundation/zebra/pull/8857/commits/5043adf9e97e1e5e872ed77197b27e454f17efa5.